src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/SqsIntegrationTestBase.kt

```diff
@@ -42,6 +42,7 @@ class SqsIntegrationTestBase : IntegrationTestBase() {
 
   protected val auditQueue by lazy { hmppsQueueService.findByQueueId("audit") as HmppsQueue }
   protected val incentivesQueue by lazy { hmppsQueueService.findByQueueId("incentives") as HmppsQueue }
```

Comment:
if i'm reading this correctly, we can remove `incentivesQueue` and use `testDomainEventQueue` in its place in tests? because ever usage of this property just gets the SQS client but then reads messages from the new `testDomainEventQueue.queueUrl`

Reply:
yes